crates/bevy_core_pipeline/src/core_2d/mod.rs

```diff
@@ -64,7 +64,7 @@ impl Plugin for Core2dPlugin {
             );
 
         let pass_node_2d = MainPass2dNode::new(&mut render_app.world);
-        let tonemapping = TonemappingNode::new(&mut render_app.world);
+        let tonemapping = ViewQueryNodeRunner::new(TonemappingNode::new(), &mut render_app.world);
```

Comment:
I am not a fan of how this reads.
We could just return a `ViewQuerryNodeRunner` from the `new` method.
So `TonemappingNode::new(&mut render_app.world)` would just return a `ViewQuerryNodeRunner<TonemappingNode>`.

Reply:
Or something like `ViewQuerryNodeRunner<TonemappingNode>::new(&mut render_app.world` to make the wrapping more explicit. This could also be automatically implemented with a default or from world impl for the `ViewNode`.

Reply:
Yeah, that's the direction I want to go. Every node is already created with a new fn that is essentially a from_world so I'm heavily considering just enforcing it for the ViewNode

Reply:
So, after playing with this. It would require either a bunch of empty FromWorld impl on ViewNodes or use the Default trait instead and lose the ability to create a node with World data.

Both approaches feel pretty subpar honestly.

Reply:
I ended up using the Default impl for FromWorld and now use `ViewNodeRunner::<Node>::from_world(&mut world)` everywhere. It's still a bit verbose but it does the job.